DataBase001/Model/City.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DataBase001.Model
+{
+    public class City : Entity
+    {
+        public int population;
```

Comment:
Строго говоря популяция это uint, потому что не может быть отрицательной популяции. По-хорошему в этом классе следовало бы либо сделать правильный тип, либо сделать свойство с сеттером { set } и проверкой на возможные значения. Вообще, считается правильным делать сразу свойства, и по правилам языка именование их идёт с большой буквы, а с маленькой чаще именуются приватные члены классов.
И вообще, его можно сделать свойством только для чтения.

```suggestion
        public readonly uint Population;
```